src/repository/opamRepository.mli

```diff
@@ -36,6 +36,14 @@ val pull_tree:
   ?subpath:string -> dirname -> OpamHash.t list -> url list ->
   string download OpamProcess.job
 
+val pulls_tree:
```

Comment:
add doc / more explicit name ? It's not clear what that `s` means.

Reply:
it need another name, if you have ideas...

Reply:
`pull_multiple_trees` ? `pull_tree_multiple` ? what's most bothering me is that where it is placed, the `s` doesn't seem like a plural.

Reply:
`pulls` is still not a plural :laughing:

Reply:
oh, i forgot about this one. Maybe `pull_shared_tree`? but it is not exactly that...